cg/clients/chanjo2/api.py

```diff
@@ -0,0 +1,40 @@
+"""Coverage analysis API for human clinical sequencing data."""
+
+import logging
+from typing import Any
+
+import requests
+
+from cg.clients.chanjo2.models import CoverageData, CoverageRequest
+
+LOG = logging.getLogger(__name__)
+
+
+class Chanjo2APIClient:
+    """
+    Chanjo2 API to communicate with a d4tools software in order to return coverage
+    over genomic intervals.
+    """
+
+    def __init__(self, config: dict[str, str]):
```

Comment:
Don't pass the config like this in the constructor, provide what is needed explicitly so it is clear what the client needs. Seems like it needs a `base_url` or whatever `host` refers to 😄

Reply:
makes sense, thanks!